frontend/src/components/inputs/LoginCard.tsx

```diff
@@ -0,0 +1,23 @@
+import { Card, Text, Stack } from "@mantine/core";
+
+export function LoginCard(props: { children: React.ReactNode }) {
+    return (
+        <Card padding="md" radius="md" withBorder>
+            <Stack justify="center" align="center" mt="xs" mb="xs" gap="sm">
+                <Text fw={600} size="lg">
+                    Please login to submit data.
+                </Text>
+                <Text size="sm" c="dimmed">
+                    This is just to stop malicious actors from submitting bad
+                    data.
+                </Text>
+                <Text size="sm" c="dimmed">
+                    We do not store anything except the UUID provided by
+                    LinkedIn.
```

Comment:
I think it would be cheeky to also include "(PROOF)" with a hyperlink to the file in the repository that handles authentication, since it's open source and all. I've never seen a privacy disclaimer with evidence in it, so the novelty of that alone is interesting to me.

Reply:
That is an interesting idea for sure, although I doubt anyone would look at the code properly enough. Either way its cool so its worthwhile to note this down somewhere.

Reply:
It'll take like 5 seconds to do, I'll add it after this PR is merged